.github/workflows/testground.yml

```diff
@@ -0,0 +1,59 @@
+name: Run Testground
+
+on:
+  push:
+jobs:
+  run-testground:
+      runs-on: ubuntu-latest
+      steps:
+      - uses: actions/setup-go@v3
+        with:
+          go-version: "^1.19.0"
+      # Get testground and our test plan code
+      - uses: actions/checkout@v3
+        with:
+         repository: 'statechannels/testground'
+         path: "code/testground"
+      - uses: actions/checkout@v3
+        with:
+          repository: 'statechannels/go-nitro-testground'
+          path: "code/go-nitro-testground"
+          ref: update-to-main #TODO Remove when https://github.com/statechannels/go-nitro-testground/pull/103 is merged
+      
+      # Set up caching for docker (which testground uses) and go modules
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
+          restore-keys: |
+            ${{ runner.os }}-go-
+      - uses: satackey/action-docker-layer-caching@v0.0.11
+        continue-on-error: true
+
+      # Build the testground daemon
+      - name: Build
+        run: make install
+        working-directory: "code/testground"
+      # Update our test plan so it uses the version of go-nitro from this workflow
+      - name: Update Test Dependency
+        run: go get github.com/statechannels/go-nitro@${{github.sha}}
+        working-directory: "code/go-nitro-testground"
+
+      - name: Import Test 
+        run:    testground plan import --from ./go-nitro-testground
+        working-directory: "code"
+      # Run a short test using the wait flag so we block until it completes
+      - name: Run Test 
+        run: | 
+         testground --endpoint=http://34.168.92.245:8042  run s --wait \
+         -p=go-nitro-testground -t=virtual-payment \
+         -b=docker:go -r=local:docker \
+         -tp=numOfHubs=2 -tp=numOfPayers=2 -tp=numOfPayees=2  -i=6 \
+         -tp=paymentTestDuration=60  -tp=concurrentPaymentJobs=1 \
```

Comment:
I think this duration is probably ok for starters. I feel like we get a good idea of performance even over 30s. I expect we will have a minute or two of setup time before we get to run the test, anyway?